app/controllers/album.js

```diff
@@ -0,0 +1,14 @@
+const fetch = require('node-fetch');
+
+const listAlbums = (req, res) => {
+  fetch('https://jsonplaceholder.typicode.com/albums')
```

Comment:
Make a service for jsonplaceholder API, also add a environment variable for external URL service (check config/index.js)